velox/exec/HashProbe.cpp

```diff
@@ -98,33 +124,43 @@ HashProbe::HashProbe(
           "HashProbe"),
       outputBatchSize_{driverCtx->queryConfig().preferredOutputBatchSize()},
       joinType_{joinNode->joinType()},
+      joinBridge_(operatorCtx_->task()->getHashJoinBridgeLocked(
+          operatorCtx_->driverCtx()->splitGroupId,
+          planNodeId())),
+      spillConfig_(makeOperatorSpillConfig(
+          *operatorCtx_->task()->queryCtx(),
```

Comment:
I wonder if we could remove this parameter from makeOperatorSpillConfig? Since we also pass operatorCtx, the queryCtx can be extract from it. This would make this code a bit shorter and easier to read.

Also, wondering if it makes sense to add operator ID to operatorCtx and make makeOperatorSpillConfig a method of OperatorCtx:

```
spillConfig_{operatorCtx_->makeSpillConfig(core::QueryConfig::kJoinSpillEnabled)}
```

Reply:
https://github.com/facebookincubator/velox/pull/2917 does this refactoring.